spec.md

```diff
@@ -734,18 +717,24 @@ float(3) / 2                                    # 1.5
 A string is an immutable sequence of elements that encode Unicode text.
 The [type](#type) of a string is `"string"`.
 
-Depending on the implementation, the elements may be 8- or 16-bit values,
-and may hold arbitrary values of the element type;
-we call the implementation's element width K.
-By convention, however, strings are typically used to hold valid encodings
-of Unicode code points, encoded using UTF-K.
+The number of bits in each string element, which we call K, is not defined by this spec,
```

Comment:
"is not defined by this spec" --> "is specified to be either 8 or 16, depending on the implementation". Avoid the implication that 8 or 16 is a suggestion and not a mandate.

Reply:
Now:
"""
For reasons of efficiency and interoperability with the host language,                                                        
the number of bits in each string element, which we call K,                                                                   
is specified to be either 8 or 16, depending on the implementation.
"""